pkg/util/filterutil/filterutil.go

```diff
@@ -542,3 +731,13 @@ func toAllocationAliasOp(labelName string, filterValue string, wildcard bool) *a
 		},
 	}
 }
+
+// GetList provides a list of values from the first key if they exist, otherwise, it returns
+// the values from the second key.
+func GetList(primaryKey, secondaryKey string, qp mapper.PrimitiveMapReader) []string {
```

Comment:
Where is this function called? I dont see it used anywhere Maybe im blind but thought of asking!

Reply:
It's used in `AssetFilterFromParamsV1`.